kernels/volk/volk_32f_8u_polarbutterfly_32f.h

```diff
@@ -246,7 +246,6 @@ static inline void volk_32f_8u_polarbutterfly_32f_u_avx(float* llrs,
         memcpy(u_temp, u + u_num - stage_size, sizeof(unsigned char) * stage_size);
 
         if (stage_size > 15) {
-            _mm256_zeroupper();
```

Comment:
This would actually be a place where zeroupper makes sense. It is followed by a function that uses SSSE3 instructions.